R/jira.R

```diff
@@ -22,39 +20,36 @@
 #' @param reporter_name name of reporter of issue (e.g. "Jane Doe")
 #' @param assignee_name name of person the issue is being assigned to (e.g. "Joe Schmo")
 #' @param comments character vector where each element is a comment string (e.g. c("This is first comment", "This is second comment"))
```

Comment:
Make sure you add @param for all your parameters so that unit checks will pass. You are missing a few here